src/RefractAPI.cc

```diff
@@ -323,13 +323,18 @@ namespace drafter {
         // ideal solution is add warning into collection
         // but there is no way how to do it
         // in current time we solve it by rethrow
-
         catch (snowcrash::Error& e) {
-            e.location = payload.sourceMap->sourceMap;
-            throw e;
+            context.warnings.push_back(
+                snowcrash::Warning("Unable to render JSON/JSONSchema: " + e.message,
```

Comment:
Please use standard format for warnings and errors. Everything in small letters. And use `.` instead of `:`.

Yes, this is something that we are told to follow long ago.